test/DefaultGroupTest.py

```diff
@@ -0,0 +1,94 @@
+'''
+Created on 04.08.2017
+
+@author: gregor
+'''
+import unittest
+import pandas as pd
+from ipet.evaluation.IPETEvalTable import IPETEvaluation, IPETEvaluationColumn,\
+    StrTuple
+import numpy
+class HelperExperiment:
+    """An experiment with data
+    """
+    d=pd.DataFrame([
+                    ["A1", "B1", 1, "ok"],
+                    ["A2", "B1", 2, "ok"],
+                    ["A1", "B2", 3, "ok"],
+                    ["A2", "B2", 4, "ok"],
+                    ], columns=["A", "B", "C", "Status"])
+    
+    def getJoinedData(self):
+        return self.d
+    
+
+
+class DefaultGroupTest(unittest.TestCase):
+    """
+    Default groups should be computed if and only if
+    a wrong user input specifies a non-existing
+    group
```

Comment:
The computation of an automatic default group based on the number of occurrences is mostly arbitrary because usually, all columns have the same number of occurrences